sw/device/lib/dif/dif_rv_core_ibex.c

```diff
@@ -124,7 +125,7 @@ dif_result_t dif_rv_core_ibex_configure_addr_translation(
   uint32_t mask = to_napot(addr_map.matching_addr, addr_map.size);
```

Comment:
what should be the right way to handle it if we already have address translate enabled, and we need to now update the address information..? could it affect your current execution?

Reply:
Yes, it could and I don't think there is a right way, i.e. we just need to be careful about our usage of slots and do not modify a slot that is currently in use like we do in ROM and ROM_EXT.

Reply:
that sounds good to me. i guess current slot is always "fixed" in a way.

Reply:
Yes, actually, the pattern that this test uses is pretty contrived. For our use cases I'd expect these translation mappings to be configured outside the lifetime of a program, e.g. before jumping to the next stage. That being said, I don't think we can guarantee anything without `fence.i`.